agent/go-service/common/closegame/register.go

```
@@ -0,0 +1,7 @@
+package closegame
+
+import maa "github.com/MaaXYZ/maa-framework-go/v4"
+
+func Register() {
+	maa.AgentServerRegisterCustomAction("CloseGameAction", &CloseGameAction{})
```

golangci-lint (2.12.2)

[error] Error return value of `maa.AgentServerRegisterCustomAction` is not checked

(errcheck)